libraries/logger/shell.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * @license
+ * This file is part of Bott.
+ *
+ * This project is dual-licensed:
+ * - Non-commercial use: AGPLv3 (see LICENSE file for full text).
+ * - Commercial use: Proprietary License (contact D@nielLaCos.se for details).
+ *
+ * Copyright (C) 2025 DanielLaCos.se
+ */
+
+/**
+ * Shell logger wrapper that calls the bash logging utilities
+ * Provides integration between TypeScript and bash log.sh
+ */
+
+import { fromFileUrl } from "@std/path";
+
+const SCRIPT_PATH = fromFileUrl(
+  new URL("../../scripts/utils/log.sh", import.meta.url),
```

Comment:
nah, i meant like...

"imports":
   "logger.sh": "./scripts/utils/log.sh"

and then...

import loggerSh from "logger.sh" with { type: "text" }